msp/src/app/modules/account/pages/child-info/child-info.component.ts

```diff
@@ -277,7 +277,11 @@ export class ChildInfoComponent extends BaseForm implements OnInit, AfterViewIni
           // Filled out "Date studies will finish"
           && this.isSet(addedChild.studiesFinishedDate)
           // Ticked "reside in BC after completing study in this school?"
-          && this.isSet(addedChild.inBCafterStudies) && addedChild.inBCafterStudies === true
+          && this.isSet(addedChild.inBCafterStudies);
+        
+        if (addedChild._status !== StatusInCanada.TemporaryResident) {
```

Comment:
Usually in JavaScript, variables that begin with an underscore ("_status") traditionally would be considered "private". If it's too large of a change to make it "public", then I wouldn't worry about it.